ltrace/loong.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/PKGBUILD b/PKGBUILD
+index a568f2f..e99d606 100644
+--- a/PKGBUILD
++++ b/PKGBUILD
+@@ -15,6 +15,17 @@ backup=('etc/ltrace.conf')
+ source=(https://sources.archlinux.org/other/$pkgname/$pkgname-$pkgver.tar.bz2)
+ sha256sums=('0e6f8c077471b544c06def7192d983861ad2f8688dd5504beae62f0c5f5b9503')
+ 
++depends+=('autoreconf')
```

Comment:
Do not add this.

1. `autoreconf` is not a `pkgname` and cannot be installed. (It's `autoconf`)
2. You just run `autoreconf` to make the pkg. It's not a runtime dependency
3. Moreover, `autoconf` is in the `depends` of `base-devel`, so it's already in the build environment